test/emqx_auth_redis_SUITE.erl

```diff
@@ -69,21 +69,38 @@ set_special_configs(_App) ->
     ok.
 
 init_redis_rows() ->
-    {ok, Connection} = ?POOL(?APP),
+    % {ok, Connection} = ?POOL(?APP),
     %% Users
-    [eredis:q(Connection, ["HMSET", Key|FiledValue]) || {Key, FiledValue} <- ?INIT_AUTH],
-
-    %% ACLs
-    emqx_modules:load_module(emqx_mod_acl_internal, false),
-    Result = [eredis:q(Connection, ["HSET", Key, Filed, Value]) || {Key, Filed, Value} <- ?INIT_ACL],
+    % [eredis:q(Connection, ["HMSET", Key|FiledValue]) || {Key, FiledValue} <- ?INIT_AUTH],
+    {ok, Server} = application:get_env(?APP, server),
+    Result = case proplists:get_value(type, Server) of
```

Comment:
Could you encapsulate these three case statements into a common method? 
It's a little more straightforward

Reply:
Sure that is a nice idea. Now I've added a method `q/1` which only takes a command as an argument.

Reply:
Cool